code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -210,7 +210,6 @@
 	description = "A mixture of milk and.... cola? Who the fuck would do this?"
 	color = "#EAC7A4"
 	taste_description = "dairy and caffeine"
-	glass_icon_state = "glass_cilk"
```

Comment:
what's happening here?

Reply:
removed the cilk sprite because it was just colored liquid in a glass, it has a reagent color so it'll look im pretty sure identical or at least very similar.